tasks/create.yaml

```
@@ -97,7 +101,12 @@ tasks:
           architecture: ${ZARF_ARCHITECTURE}
       - description: "Create the shim bundle (Base only)"
         if: ${{ eq .inputs.layer "base" }}
-        cmd: "uds create bundles/base-shim --confirm --no-progress --architecture=${ZARF_ARCHITECTURE}"
+        cmd: |
+          if [ ${{.inputs.shim_bundle}} = "true" ]; then
+            uds create bundles/base-shim --confirm --no-progress --architecture=${ZARF_ARCHITECTURE}
+          else
+            echo "Skipping base-shim bundle creation."
+          fi
```

nit: when adding script logic like this it might be good idea to specify the shell like so we get consistency across envs
```suggestion
        cmd: |
          if [ ${{.inputs.shim_bundle}} = "true" ]; then
            uds create bundles/base-shim --confirm --no-progress --architecture=${ZARF_ARCHITECTURE}
          else
            echo "Skipping base-shim bundle creation."
          fi
        shell:
          linux: bash
          darwin: bash
```

---

I had been under the impression that the default shell is `sh` when not specified based on [Zarf docs](https://docs.zarf.dev/ref/actions/#cmd-action-configuration) (which these actions are based on/share a common library for). When I added this I made sure that it was `sh` specific (single `=` and `[]`) and tested locally (mac) plus in CI (linux).

 I did go ahead and set this to `bash` and update the conditional as well (to use a more specific bash format) anyways - but I think this might be okay either way?

---

ahh yeah that is true. i just saw the commit for the wrong shell logic and thought you might have run into an issue where it wasn't working locally for you.  it should be fine either way.

---

So I do think bash is a safer option here. That commit was because locally it worked but in CI it failed (technically worked, but evaluation of the conditional threw an error log). It seems like that's due to a difference in `sh` on Linux/Mac ([ref](https://stackoverflow.com/questions/19943767/why-does-bin-sh-behave-differently-on-mac-and-ubuntu)). I've definitely experienced this in the past where certain things work locally on `sh` for me (on my Mac) but definitely are not valid in dash on my Ubuntu machine. `bash` should hopefully be a more standard experience for the most part.